src/ServiceControl.Audit/Infrastructure/Settings/Settings.cs

```diff
@@ -109,6 +109,7 @@ public string RootUrl
         public int Port { get; set; }
 
         public bool PrintMetrics => SettingsReader.Read<bool>(SettingsRootNamespace, "PrintMetrics");
+        public string OtelMetricsUrl { get; set; } = SettingsReader.Read<string>(SettingsRootNamespace, nameof(OtelMetricsUrl));
```

Comment:
See other comment on `OTEL_EXPORTER_OTLP_ENDPOINT`

```suggestion
        const string OTEL_EXPORTER_OTLP_ENDPOINT = nameof(OTEL_EXPORTER_OTLP_ENDPOINT);
        []
        public string OtelExporterOltpEndpointUrl { get; init; } = SettingsReader.Read<string>(SettingsRootNamespace, OTEL_EXPORTER_OTLP_ENDPOINT);
```